app/controllers/users_controller.rb

```diff
@@ -18,8 +25,12 @@ def accept_terms
 
 private
 
+  def user
+    @user ||= User.find(current_user["user_id"]).first
+  end
+
   def accept_screen(method, path)
-    User.member(current_user["user_id"]).send(method)
+    user.send(method)
```

Comment:
Not a blocker for this PR, but it'd be a good habit to start to change these to `__send__` as we encounter them.

```suggestion
    user.__send__(method)
```

Reply:
👍